app/controllers/JourneyController.scala

```diff
@@ -132,8 +132,11 @@ class JourneyController @Inject() (
       passcode =>
         emailVerificationConnector.validatePasscode(journeyId, passcode).map {
           case ValidatePasscodeResponse.Complete(redirectUri) =>
+            val isLocal: Boolean = appConfig.platformFrontendHost.isEmpty
```

Comment:
Is this config value only used to determine this isLocal flag? if so might it be better as a boolean, like "allowRelativeUrls" or similar?